core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -656,6 +657,7 @@ public Map<String, SaltPillarProperties> createPillarWithClouderaManagerSettings
                                 "disable_auto_bundle_collection", disableAutoBundleCollection,
                                 "set_cdp_env", isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_0_2),
                                 "cloud_provider_setup_supported", isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_6_2),
+                                "is_cloud_provider_type_supported", isCloudProviderTypeSupported(stackDto),
```

Comment:
maybe the value of `cloud_provider_setup_supported` could be the combined result of 2 checks. what is the added value pushing this new pillar?